Platformio/HAL/Targets/ESP32/HardwareRevX.cpp

```diff
@@ -304,19 +278,6 @@ void HardwareRevX::setupIMU() {
   IMU.readRegister(&intDataRead, LIS3DH_INT1_SRC); // clear interrupt
 }
 
-// TODO move to display
-void HardwareRevX::slowDisplayWakeup() {
```

Comment:
I do wonder if we may still need this it would probably be good to get Max's input on the hardware.

Reply:
I moved it over. Since the HAL can handle diff hardware versions we could tie to an Env var. I wonder if Max is the only one with hardware that needs it.